jax/_src/test_util.py

```diff
@@ -1142,7 +1142,8 @@ def floating(self):
 
   @_cached_property
   def all_floating(self):
-    return self.supported([_dtypes.bfloat16, np.float16, np.float32, np.float64])
+    return self.supported([_dtypes.float8_e4m3fnuz, _dtypes.float8_e5m2fnuz,
```

Comment:
This would affect a lot of tests: let's avoid adding 8-bit floats to `all_floating` due to them not being compatible with implicit type promotion, and being incompatible with many operations that JAX supports.